site/content/3.13/graphs/graph-visualizer.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Graph Visualizer
+menuTitle: Graph Visualizer
+weight: 102
+description: >-
+  Explore your ArangoDB graphs with an intuitive and interactive interface
```

Comment:
I'd suggest something like: Visually explore and interact with your ArangoDB graphs through an intuitive web interface